code_generation/producers/scalefactors.py

```diff
@@ -57,3 +87,101 @@
         ],
     },
 )
+MuonIDIso_SF_old = ProducerGroup(
+    name="MuonIDIso_SF_old",
+    call=None,
+    input=None,
+    output=None,
+    scopes=["mt", "em", "mm"],
+    subproducers={
+        "mt": [
+            Muon_1_ID_SF_old,
+            Muon_1_Iso_SF_old,
+        ],
+        "em": [
+            Muon_2_ID_SF_old,
+            Muon_2_Iso_SF_old,
+        ],
+        "mm": [
+            Muon_1_ID_SF_old,
+            Muon_1_Iso_SF_old,
+            Muon_2_ID_SF_old,
+            Muon_2_Iso_SF_old,
+        ],
+    },
+)
+
+############################
+# Tau ID/ISO SF
+# The readout is done via correctionlib
+############################
+Tau_1_VsJetTauID_SF = ExtendedVectorProducer(
+    name="VsJetTauID_SF",
+    call='scalefactor::tau::id({df}, {input}, {vec_open}{tau_dms}{vec_close}, "{vsjet_tau_id_WP}", "{tau_sf_variation}", "{tau_sf_dependence}", {output}, "{tau_sf_file}", "DeepTau2017v2p1VSjet")',
```

Comment:
`DeepTau2017v2p1VSjet` make this a config parameter

Reply:
Done